spark-operator.advisories.yaml

```
@@ -0,0 +1,67 @@
+schema-version: "2"
+
+package:
+  name: spark-operator
+
+advisories:
+  - id: CVE-2023-2431
+    events:
+      - timestamp: 2023-09-07T13:45:11Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-in-execution-path
+          note: Vulnerable code is part of the Kubernetes kubelet which is not in the execution path of the spark-operator
+
+  - id: CVE-2023-2727
+    events:
+      - timestamp: 2023-09-07T13:47:12Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: Vulnerable code is part of the Kubernetes API server and not importable.
+
+  - id: CVE-2020-8554
+    events:
+      - timestamp: 2023-09-07T13:49:44Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-in-execution-path
+          note: This is a Kubernetes API flaw, and this code is not reachable in our package.
+
+  - id: CVE-2023-2728
+    events:
+      - timestamp: 2023-09-07T13:53:32Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: Vulnerable code is part of the Kubernetes ServiceAccount admission plugin which is not importable
+
+  - id: CVE-2020-8564
+    events:
+      - timestamp: 2023-09-07T13:55:27Z
+        type: true-positive-determination
+        data:
+          note: Pending upstream project to pick up k8s.io/kubernetes 1.20.0-alpha.1+.
+
+  - id: CVE-2020-8565
+    events:
+      - timestamp: 2023-09-07T13:59:27Z
+        type: true-positive-determination
+        data:
+          note: Pending upstream project to pick up k8s.io/kubernetes 1.20.0-alpha.1+.
```

It looks like you might've changed this one too, but the comment to change to `...alpha.1` was for the `CVE-2020-8564` advisory.

The fix for `CVE-202-8565` is still marked as `...alpha.2`: https://github.com/golang/vulndb/blob/6055ff05e0a2e13ed88d85ad98eb79ed2883ec0c/data/reports/GO-2021-0064.yaml#L5

```suggestion
          note: Pending upstream project to pick up k8s.io/kubernetes 1.20.0-alpha.2+.
```

---

oh nice, thank you, yes.